pybamm/parameters/sei_parameters.py

```diff
@@ -51,25 +59,26 @@
 L_x = pybamm.standard_parameters_lithium_ion.L_x
 
 i_typ = pybamm.electrical_parameters.i_typ
-j_scale_n = pybamm.standard_parameters_lithium_ion.interfacial_current_scale_n
-j_scale_p = pybamm.standard_parameters_lithium_ion.interfacial_current_scale_p
 
+j_scale = st.interfacial_current_scale_n
 
-C_sei_reaction_n = (j_scale_n / m_sei_dimensional) * pybamm.exp(
-    -(F * U_n_ref / (2 * R * T_ref))
-)
-C_sei_reaction_p = (j_scale_p / m_sei_dimensional) * pybamm.exp(
-    -(F * U_n_ref / (2 * R * T_ref))
-)
+C_sei_reaction = (
+    F * L_sei_0_dim / (m_sei_dimensional * tau_discharge * V_bar_inner_dimensional)
+) * pybamm.exp(-(F * U_n_ref / (2 * R * T_ref)))
```

Comment:
please fix the changes overwritten in this file

Reply:
Done.